resources/qml/PrintSetupHeaderButton.qml

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.1
+import QtQuick.Controls 2.1
+
+import Cura 1.0 as Cura
+import UM 1.5 as UM
+
+ToolButton
+{
+    id: base
+
+    property alias tooltip: tooltip.text
+
+    contentItem: Label {}
+
+    Cura.ToolTip
+    {
+        id: tooltip
+        visible: base.hovered
+        targetPoint: Qt.point(parent.x, Math.round(parent.y + parent.height / 2))
+    }
+
+    background: Rectangle
+    {
+        color:
```

Comment:
Depending on how much time we have left; I think that it's better if we handle this with states (although I'm okay if it's not done. Left up to the implementer to make that choice!)

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/a2d03af2f147bf0c073472fe694d6d14f7ea98a1